bin/clever.js

```diff
@@ -578,7 +578,7 @@ function run () {
   const activity = lazyRequirePromiseModule('../src/commands/activity.js');
   const activityCommand = cliparse.command('activity', {
     description: 'Show last deployments of an application',
-    options: [opts.alias, opts.follow, opts.showAllActivity],
+    options: [opts.alias, opts.follow, opts.showAllActivity, opts.humanJsonOutputFormat],
```

Comment:
issue: I know you don't agree with @miton18 (and maybe me) on this but in logs and access-logs commands, `json` format produces valid JSON and not NDJSON. I know `jq` will handle both but I think we should discuss the naming.

Also, unlike logs and access-logs that can stream and end, `clever activity -f` will virtually never end.

Reply:
I am very open to discussion on that point.